staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -62,14 +62,13 @@ var (
 	responseSizesSummary = prometheus.NewSummaryVec(
 		prometheus.SummaryOpts{
 			Name: "apiserver_response_sizes_summary",
-			Help: "Response size summary in bytes for each verb, resource, subresource and namespace.",
+			Help: "Response size summary in bytes for each verb, resource, subresource and if it's namespaced.",
 			// Make the sliding window of 1h.
 			MaxAge: time.Hour,
 		},
-		[]string{"verb", "resource", "subresource", "namespace"},
+		[]string{"verb", "resource", "subresource", "namespaced"},
```

Comment:
Changed it everywhere except in installer.go where "scope" is already being used by restmapper's scope.